js/languages/en.js

```diff
@@ -389,6 +389,8 @@ export default {
     'ending-short': '{T} left',
     long: '{T} long',
     over: "School's over!",
+    range: '{T1}–{T2} · {D}',
```

Comment:
This is only used by periods, not events (because events may or may not have a timestamp or location)